tests/JestExt.test.ts

```diff
@@ -27,6 +27,8 @@ import { updateCurrentDiagnostics } from '../src/diagnostics'
 import { JestProcessManager, JestProcess } from '../src/JestProcessManagement'
 import * as messaging from '../src/messaging'
 
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "expectItTakesNoAction"] }] */
+
```

Comment:
We might be able to do this "clean"-ly using a custom matcher -- `expect.extend(fn)`,  https://jestjs.io/docs/en/expect#expectextendmatchers. I'd keep it as is for now.

Reply:
agree, but figure it might not be worthy to do it just for one test file, would be a fun exercise for the future PR.